index.html

```diff
@@ -1159,6 +1220,10 @@ <h2>
         "DOM#concept-event-fire">fire an event</dfn>, <dfn data-cite=
         "DOM#interface-eventtarget">EventTarget</dfn>
       </p>
+      <p>
+        The following is defined in [[WEBIDL]]: <dfn data-cite=
+        "WEBIDL#idl-promise">promise</dfn>
```

Comment:
I had to add this definition of promise into dependencies, not sure if that was the right thing to do?

Reply:
This is great. We talk about promises a lot, so this is helpful.